### PR TITLE
Catalog: Unit tests

### PR DESCRIPTION
**Pull request by mateo951**
- Added unit tests for genre and albums

### CODE REVIEW

File: lib/spec/album_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+describe MusicAlbum do
+  context 'When testing creation of Album instance' do
+    it 'Should create a new person when we call the class constructor' do
+      new_album = MusicAlbum.new(true, 2)
+      expect(new_album.on_spotify).to eq true
+      expect(new_album.published_date).to eq 2
+    end
+  end
+  context 'When testing the Album methods' do
```
Comment (Felipeg005) [inline lib/spec/album_spec.rb:12]: Great Job with all unit testing 👍 As optional, remember that the last reviewer asks us to add more descriptions and separate the creation of the class from the methods.

File: lib/spec/genre_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    end
+  end
+  context 'When testing genre methods' do
+    it 'Should add a new item to the items array' do
+      new_genre = Genre.new('Test')
+      item = Item.new(2)
+      new_genre.add_item(item)
+      expect(new_genre.items.length).to eq 1
+    end
+  end
```
Comment (Felipeg005) [inline lib/spec/genre_spec.rb:19]: Well done 😎

**Review by Felipeg005 — APPROVED** [review]
Hello @mateo951 
Great Job with this implementation!!!

## Approved✔️ 

Time to merge this branch. 😎 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._***

File: lib/spec/album_spec.rb (hunk 1 above)
Comment (mateo951) [reply lib/spec/album_spec.rb:12]: Thanks! I'll listen to your feedback

**Pull request merged**